DemoApp/DemoApp/Views/MultiLineInputPage.xaml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage x:Class="DemoApp.MultiLineInputPage"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:Modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             xmlns:constants="clr-namespace:DemoApp.Constant"
+             Title="MultiLineInput">
+  <ScrollView>
+    <VerticalStackLayout>
+
+      <StackLayout Padding="10"
+                   Spacing="10">
+
+        <StackLayout Orientation="Horizontal">
+          <Label Grid.Column="0"
+                 Text="IsEnabled"
+                 HorizontalOptions="Center"
+                 VerticalOptions="Center" />
+          <Switch Grid.Column="1"
+                  x:Name="isEnabled"
+                  IsToggled="{Binding IsEnabled}"
+                  Toggled="IsEnabled_Toggled" />
+          <Label Grid.Column="0"
+                 Text="IsReadOnly"
+                 HorizontalOptions="Center"
+                 VerticalOptions="Center" />
+          <Switch Grid.Column="1"
+                  x:Name="isReadOnly"
+                  IsToggled="{Binding IsReadOnly}"
+                  Toggled="IsReadOnly_Toggled" />
+          <Label Grid.Column="0"
+                 Text="IsAutoSize"
+                 HorizontalOptions="Center"
+                 VerticalOptions="Center" />
+          <Switch Grid.Column="1"
+                  x:Name="IsAutoSize"
+                  IsToggled="{Binding IsAutoSize}"
+                  Toggled="IsAutoSize_Toggled" />
+        </StackLayout>
+
+        <Modus:MultiLineInput  x:Name="modusInput"
+                           Placeholder="Text Input"
+                           AutoSize="{Binding IsAutoSize}"
+                           IsEnabled="{Binding IsEnabled}"
+                           IsReadOnly="{Binding IsReadOnly}"/>
+
+        <Modus:MultiLineInput  x:Name="modusInputWithTitle"
+                           TitleText="MultiLine Input"
+                           AutoSize="{Binding IsAutoSize}"
+                           IsEnabled="{Binding IsEnabled}"
+                           IsReadOnly="{Binding IsReadOnly}"
+                           Placeholder="Text Input" />
+
+        <Modus:MultiLineInput TitleText="Success/Error MultiLine Input"
+                          HelperText="Enter more or equal to 100 characters"
```

Comment:
Please set the valiation to set 10 characters. 100 is too large to verify

Reply:
Changed the validation threshold to 10